src/services/dex.js

```diff
@@ -2888,4 +2888,19 @@ export default {
     // console.log(`${JSON.stringify(res)}`);
     return res.data.kycStatus;
   },
+
+  getMarketDataFromName(marketName) {
+    let marketData = false;
+    store.state.markets.forEach((market) => {
```

Comment:
Could use lodash here for brevity `_.find`

Reply:
Updated to use lodash.